lib/internal/event_target.js

```diff
@@ -366,15 +366,6 @@ ObjectDefineProperties(CustomEvent.prototype, {
   detail: kEnumerableProperty,
 });
 
-class NodeCustomEvent extends Event {
```

Comment:
If you keep this as `class NodeCustomEvent extends CustomEvent {}`, that might even be a bit nicer since it’s a way to differentiate between events coming from a user and events coming from a Node.js EventEmitter `.emit()` call?

Reply:
Could you help me understand what specific usercase do we need to support? I don't understand in which scenario  we would need to differentiate between events coming from a user and events coming from a Node.js EventEmitter .emit() call.